app/src/main/res/layout/rsb_view_submitbar.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
It looks like this actually replaces the entire app's submit bar.  Which, is awesome for the sake of CRF, but I was looking to have this be more encapsulated.  Can you move this to crf_view_submitbar.xml and then have CrfFormStepLayout override public @LayoutRes int getFixedSubmitBarLayoutId() {}?